tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/manifest.ini

```diff
@@ -10,3 +10,9 @@ smoketest = true
 stable = false
 
 [test_clock_set_alarm.py]
+
+[test_clock_run_stopwatch_laps.py]
+skip-if = device == "desktop"
```

Comment:
This line was added, not because it failed, but because it's a dogfood test and we don't run those on device. I will remove the statement though.